### PR TITLE
Update README.md with instructions for .env keys

### PR DESCRIPTION
**Pull request by scotthallock**
### Overview

Having gone through this process myself to run the c0d3 bot locally, I thought it would be nice to update the README to include instructions on where to find the several `.env` keys required to start the bot.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,15 +12,48 @@ The production bot runs on the c0d3 CapRover server and can be communicated with
...
+First you need to register a discord application to get the `BOT_TOKEN` that c0d3r will use to forward your commands to your discord server. A guide for that setup can be found on the wiki page here: [Registering a Discord Bot](https://github.com/garageScript/c0d3r/wiki/Registering-a-Discord-Bot#registering-a-discord-bot)
 
-- ## Using yarn
+## Setting the .env variables
+
+Copy and rename `example.env` to `.env` and fill the required variables below
+
+`DISCORD_TOKEN`: to find your Discord token, you can [follow these instructions](https://www.androidauthority.com/get-discord-token-3149920/).
+
+`BOT_TOKEN`: Ensure you have followed the above instructions for registering the application/bot.
```
Comment (JasirZaeem) [inline README.md:23]: This is not what BOT_TOKEN does or how you make it, check the notes below in the readme.

File: README.md [hunk 2]
```diff
@@ -12,15 +12,48 @@ The production bot runs on the c0d3 CapRover server and can be communicated with
 
 ## Development - Getting Started
 
-\* First you need to register a discord application to get the `DISCORD_TOKEN` that c0d3r will use to forward your commands to your discord server. A guide for that setup can be found on the wiki page here: [Registering a Discord Bot](https://github.com/garageScript/c0d3r/wiki/Registering-a-Discord-Bot#registering-a-discord-bot)
+First you need to register a discord application to get the `BOT_TOKEN` that c0d3r will use to forward your commands to your discord server. A guide for that setup can be found on the wiki page here: [Registering a Discord Bot](https://github.com/garageScript/c0d3r/wiki/Registering-a-Discord-Bot#registering-a-discord-bot)
```
[the remark this answers is not in the document]
Comment (scotthallock) [reply README.md:15]: Oh, I had a misunderstanding about this.

Following the instructions in the link, I did not find the DISCORD_TOKEN. What I did find, under the settings for the "Bots" tab of the selected application, was a "Reset Token" button. So I thought this must be the BOT_TOKEN.

Now I see that for testing, the BOT_TOKEN can be any non-empty string and the bot will work.

So, where does the BOT_TOKEN come from, for the production .env file?

[the remark this answers is not in the document]
Comment (flacial) [reply README.md:15]: `BOT_TOKEN` is supposed to be a secret that the client uses to access the bot. [Check this line](https://github.com/garageScript/c0d3r/blob/5d9a14a8d5b1c4196244804beae1f5f8c8c3e1bf/src/api/middlewares/authorized.ts#L5)

[the remark this answers is not in the document]
Comment (JasirZaeem) [reply README.md:15]: > So, where does the BOT_TOKEN come from, for the production .env file?

When running locally, you can set any string you want as your bot token. For the production app that .env variable is set in our CapRover server.

File: README.md [hunk 3]
```diff
@@ -12,15 +12,48 @@ The production bot runs on the c0d3 CapRover server and can be communicated with
...
 
-- ## Using yarn
+## Setting the .env variables
+
+Copy and rename `example.env` to `.env` and fill the required variables below
+
+`DISCORD_TOKEN`: to find your Discord token, you can [follow these instructions](https://www.androidauthority.com/get-discord-token-3149920/).
```
Comment (JasirZaeem) [inline README.md:21]: The discord token this article is about is unrelated to the one needed for the bot. The link above, https://github.com/garageScript/c0d3r/wiki/Registering-a-Discord-Bot#registering-a-discord-bot is how we can get the DISCORD_TOKEN

**Pull request closed without merging**